pimd/pim_cmd.c

```diff
@@ -7416,6 +7416,31 @@ DEFUN (interface_no_ip_pim_hello,
 	return pim_process_no_ip_pim_hello_cmd(vty);
 }
 
+
+DEFPY (interface_ip_pim_passive,
+       interface_ip_pim_passive_cmd,
+       "[no$no] ip pim passive",
+       NO_STR
+       IP_STR
+       PIM_STR
+       "disable exchange of protocol packets\n")
```

Comment:
nit: Could you capitalize the first word?

Reply:
Displaying passive status in show ip pim interface command